packages/blockchain-extension/extension/webview/PreReqView.ts

```diff
@@ -191,12 +191,21 @@ export class PreReqView extends View {
         let missingPrerequisitesLength: number = 0;
 
         for (const _dependency of Object.keys(dependencies)) {
+            let dependency: any = dependencies[_dependency];
```

Comment:
If you want to remove the `any` here it could be changed to `Dependency` which is in `'../dependencies/Dependencies';`